Week3/homework/3-step3.js

```diff
@@ -3,23 +3,38 @@
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
   // add your code here
-  return str;
+  let string = "";
+  for (let i = 0; i < num; i++) {
+    string += str;
+  }
+  return string;
 }
 
 console.log('for', repeatStringNumTimesWithFor('abc', 3));
 
 // use a 'while' loop
 function repeatStringNumTimesWithWhile(str, num) {
   // add your code here
-  return str;
+  while (num > 0) {
```

Comment:
You have to define string before you can use it.  
This is where your program errors out because it cannot find string.
It is very obvious that you have not *run* or *tested* your program, otherwise you would have seen the `error message`, and been able to correct the problem which is:
```
let string = '';
```